src/main/java/com/filip/workshop_managment/service/VehicleServiceImpl.java

```diff
@@ -18,21 +18,21 @@ public VehicleServiceImpl(VehicleRepository vehicleRepository) {
     }
 
     @Override
-    public List<Vehicles> listAll() {
-        List<Vehicles> vehicles = new ArrayList<>();
+    public List<Vehicle> listAll() {
+        List<Vehicle> vehicles = new ArrayList<>();
        vehicleRepository.findAll().forEach(vehicles::add);
```

Comment:
Czy findAll nie zwraca aby od razu listy albo jakiejś kolekcji? Trzeba to przepisywać?